src/saveload/marker.rs

```diff
@@ -387,8 +417,7 @@ where
 }
 
 /// Basic marker allocator, uses `u64` as identifier
-#[derive(Derivative)]
-#[derivative(Clone, Debug)]
+#[derive(Clone, Debug)]
```

Comment:
Gotta manually implement this, the `std` derive would add a `Clone` and `Debug` bound to `T`